scripts/check_msi_installation.ps1

```diff
@@ -14,7 +14,7 @@ $InstallPath = "$env:ProgramFiles\ebpf-for-windows";
 # - The expected file lists for each build artifact
 $buildArtifactParams = @{
     "Build-x64_Debug" = @{
-        "InstallComponents" = "ADDLOCAL=eBPF_Runtime_Components,eBPF_Runtime_Components_JIT"
+        "InstallComponents" = "ADDLOCAL=eBPF_Runtime_Components"
```

Comment:
github is not letting me put this comment below, but I think there is one more part below that needs updating - we check for eBPF_Runtime_Components_JIT to determine if we need to check ebpfsvc